README.md

```diff
@@ -1,21 +1,39 @@
 # RPCSX Wiki
 
-## Contributing
+## Installation
+Tested on Ubuntu WSL
+- Update your repository lists and packages:
+```
+sudo apt update && sudo apt upgrade -y
+```
 
-### Project setup
-Dependencies: Ruby 3.x
-```sh
-gem install bundler
+- Install dependencies:
+```
+sudo apt-get install ruby-full build-essential dh-autoreconf libffi-dev libssl-dev
+```
```

Comment:
This is not the recommended install method for Ruby, and has dependencies specific to your Ubuntu release number 

Please replace with rvm or a link to the rvm instructions, and add a note about the two required libraries on your version